extensions/wikia/BannerNotifications/js/BannerNotifications.js

```diff
@@ -112,6 +113,7 @@
 			this.timeout = defaultTimeout;
 		}
 
+		debugger;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this debugger statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ADebuggerStatement)